rewe.js

```diff
@@ -6,14 +6,14 @@ exports.format_csv = function(data) {
     el = data[k];
     storetype = "REWE?";
     if (el.group == "BP_") storetype = "BIPA";
-    if (el.group == "BA_") storetype = "BILLA";
-    if (el.group == "MS_") storetype = "MERKUR";
+    if (el.group == "BA_") storetype = "Billa";
+    if (el.group == "MS_") storetype = "Merkur";
     if (el.group == "PE_") storetype = "PENNY";
```

Comment:
I didn't change that because on their website they always write PENNY instead of Penny. I tried to use the origin/write naming here